src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -785,6 +785,21 @@ impl HummockEventHandler {
                         error!("unable to send get_min_uncommitted_sst_id result: {:?}", e);
                     });
             }
+            HummockEvent::RegisterVectorWriter {
+                table_id,
+                init_epoch,
+            } => self.uploader.register_vector_writer(table_id, init_epoch),
+            HummockEvent::VectorWriterSealEpoch {
+                table_id,
+                next_epoch,
+                add,
```

Comment:
Since `add` is provided here, uploader is not responsible for the actual uploading. Do we still need to make uploader be aware of vector?

Reply:
The `add` should be carried in the result of `sync`. Currently, the `tx` of `SyncResult` is passed to the uploader, and after the upload tasks of the sync have finished, inside the uploader we will send the `SyncResult` via the `tx`. Therefore it's simpler to pass the `add` to `uploader` so that it can carry the `add` in the `SyncResult`.